spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -267,3 +267,92 @@ definitions:
             desc: |
               Phase biases corrections for a
               satellite being tracked.
+
+ - MSG_SSR_ORBIT:
+    id: 0x05DB
+    short_desc: Precise orbit correction
+    desc: |
+      The precise orbit correction message is to be applied 
+      as a delta correction to broadcast ephemeris and is 
+      typically an equivalent to the 1057 RTCM message types
+    fields:
+        - time:
+            type: GPSTimeSec
+            desc: GNSS reference time of the correction
+        - sid:
+            type: GnssSignal
+            desc: GNSS signal identifier (16 bit)
+        - update_interval:
+            type: u8
+            units: s
+            desc: Update interval between consecutive corrections
+        - iod_ssr:
+            type: u8
+            desc: |
+                IOD of the SSR correction. A change of Issue Of Data
+                SSR is used to indicate a change in the SSR 
+                generating configuration
+        - iod:
+            type: u32
+            desc: Issue of broadcast ephemeris data or IODCRC (Beidou)
+        - radial:
+            type: s32
+            units: 0.1 mm
+            desc: Orbit radial delta correction
+        - along:
+            type: s32
+            units: 0.4 mm
+            desc: Orbit along delta correction
+        - cross:
+            type: s32
+            units: 0.4 mm
+            desc: Orbit along delta correction
+        - dot_radial:
+            type: s32
+            units: 0.001 mm/s
+            desc: Velocity of orbit radial delta correction
+        - dot_along:
+            type: s32
+            units: 0.004 mm/s
+            desc: Velocity of orbit along delta correction
+        - dot_cross:
+            type: s32
+            units: 0.004 mm/s
+            desc: Velocity of orbit cross delta correction
+
+ - MSG_SSR_CLOCK:
+    id: 0x05DA
+    short_desc: Precise clock correction
+    desc: |
+      The precise clock correction message is to be applied 
+      as a delta correction to broadcast ephemeris and is 
+      typically an equivalent to the 1058 RTCM message types
+    fields:
+        - time:
+            type: GPSTimeSec
+            desc: GNSS reference time of the correction
+        - sid:
+            type: GnssSignal
+            desc: GNSS signal identifier (16 bit)
+        - update_interval:
+            type: u8
+            units: s
+            desc: Update interval between consecutive corrections
+        - iod_ssr:
```

Comment:
So the problem is that the rtcm message is not stateless. the iod of the 1058 message refers to the iod of the related 1057 message previously sent:
![image](https://user-images.githubusercontent.com/22922607/57248665-cd17b900-6ff7-11e9-97d6-aa135bde40ca.png)

For SBP we would probably want to keep it as stateless as possible so I think we might want the IOD in the clock only solution. 
@benjamin0 @jbangelo  What is the context of this PR? Do we want to reduce the bandwidth and be able to send a "high rate" clock sbp message?

Reply:
@scarcanague context is supporting Geo++ SSR which is delivering clock and orbit messages independently.

Reply:
OK. I think we want to add the IOD then, that might complicate the RTCM->SBP conversion but otherwise we will run into trouble if our SBP stream does not end up being sequenced properly

Reply:
Adding in the IOD will mean that the RTCM->SBP conversion will require some additional state and logic to properly convert the messages.

If that's the route we want to take I'd argue that we shouldn't add these new messages at all and have the gnss-converters tool combine the separate orbit and clock SSR messages into our previously defined combined orbit and clock message. I think that would require similar logic/buffering in the conversion tool anyways.

Or do we want a 1-to-1 relationship between RTCM and SBP messages?

Reply:
The goal of this work is to unblock being able to test the Geo++ C&O stream. I'd be in favor of making the assumption that we won't lose or receive messages out of order for now. If we end up relying on this implementation, then we'll need to revisit this assumption.

Reply:
Seems reasonable. Should this be merged now?